x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -76,12 +80,20 @@ const CustomGridRow = styled.div.attrs<{
   border-bottom: 1px solid ${(props) => (props.theme as EuiTheme).eui.euiBorderThin};
```

Comment:
I think your issue comes from the `width: fit-content;` above this line. Your loading rows end up having 0 width so the `intersectionObserver` doesn't ending up tracking it.

Reply:
I swear I remember adding `width: 100%`, let me check again.